cime_config/testdefs/testlist_clm.xml

```diff
@@ -796,6 +796,15 @@
       <option name="comment"  >test transient PFTs (via HIST) when reading 78-pft data and running with 16 pfts</option>
     </options>
   </test>
```

Comment:
@ekluzek, do I need to use decStart with this test because it's an IHist case?

Reply:
The decStart is done so that you can run over a year end, without running through an entire year. So it depends on what you want to accomplish with the test.

Reply:
I'm testing n_dom_pfts = 2 in a transient vegetation case. The test runs for 10 days (that's what d10 means, right?). Transient vegetation changes daily (correct me if I'm wrong), so I have decided to not use decStart in this test. I'm open to differing opinions.

Reply:
Yes, _Ld10 means 10 days. Vegetation changes only happen at the end of the year for transient simulations. Some of the fluxes from those changes are dribbled out over time, but the update of PFT's happens at the end of the year. So that would be a reason to use a decStart test.

Reply:
Ok, I will add decStart to this test.

Reply:
...and to the one before it.

Reply:
When standard testing passes, I will commit and push.

Reply:
Done.